web/src/hashchange.js

```diff
@@ -185,8 +184,15 @@ function do_hashchange_normal(from_reload) {
             maybe_hide_recent_view();
             inbox_ui.show();
             break;
+        case "#global_feed":
+            show_all_message_view();
+            break;
         case "#all_messages":
+            // Since "#all_messages" was deprecated and replaced by
+            // "#global_feed" current URL hash is replaced by
+            // "#global_feed" after showing the view.
             show_all_message_view();
+            window.location.replace("#global_feed");
```

Comment:
Can you put the changes to change its hash into a separate commit?

It'll be much easier to review that way, with a commit touching a lot of files just changing strings, and then anything else in separate commits.